example/main.cpp

```diff
@@ -20,7 +21,7 @@ void createFileSink() {
 	Logger::addSink<FileSink>({ "./", "log.txt" });
 
 	FileSink::Params paramFileSink { "./", "log2.txt" };
-	paramFileSink.maxFileSize = 150;
+	paramFileSink.maxFileSize = 150_B;
```

Comment:
please increase the size to 5_MB, 150_B will store almost nothing

Reply:
done